### DISCUSSION TITLE
Rotating lamp for the DIYson express

### DISCUSSION THREAD

**Discussion by jfuchs** [stevenbennett/DIYson#42] [Ideas]
I found myself wanting to modify the DIYson express so that I could change the orientation of the lamp. Here's my latest iteration:

![image](https://user-images.githubusercontent.com/21195/235208308-b9a3ba3b-1f5b-441a-8219-db3bf70a835d.png)
![image](https://user-images.githubusercontent.com/21195/235208295-7df6eb45-284c-496f-a620-436268d46d91.png)

I'm pretty happy with it functionally, though I'd like to also take another pass at getting the aesthetic to match the rest of the lamp a bit better.

If anyone is interested, I'd be happy to share the design — either as a PR against this repo or I can create a fork (I'll defer to @stevenbennett on that)

**Comment by stevenbennett** [comment]
@jfuchs  ⭐️👍🏻 This is awesome! Super useful functionality and really nice execution! I like the clip solution you used to hold the light while still keeping the usb port exposed. Nice work :)

The original DIYson directory has had a sub-directory called "Fan-Made Alternatives" for these kinds of mods. I just added the same directory to the Express project if you'd like to add it there (see @Tauk1 's commit f031e05 as an example).

I'm also totally open to any better alternatives to sharing and storing enhancements and mods like this, so if anyone has suggestions please feel free to leave them.

Thanks for sharing, awesome addition :)

**Reply by jfuchs** [reply]
Thanks, This seems like a reasonable system — I don't know how often I'll want to iterate on this or update it, but if it ends up being frequent I may do that in my fork and then make PRs when things are stable.

The PR is up to add this variant: https://github.com/stevenbennett/DIYson/pull/43

**Comment by FinstererFred** [comment]
@jfuchs  

Great idea! I'm very much interested in the files 👍

**Reply by jfuchs** [reply]
They'll be included with https://github.com/stevenbennett/DIYson/pull/43!
